client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -0,0 +1,882 @@
+          />
+          <LessonTip>
+            Observați atributul
+            {' '}
+            <strong className="formatted">ID</strong>
+            .
+            Aceasta este modalitatea prin care facem legătura între link și
+            secțiunea paginii la care vrem să ajungem.
+          </LessonTip>
+          <p>
+            Deși nu am ajuns la partea de CSS, merită menționat că jump links
+            trebuie să beneficieze de
+            {' '}
+            <a href="/concepts/smooth-scrolling">smooth scrolling</a>
+            {' '}
+            pentru ca
+            userii să aibă o experiență plăcută. Un usage case este chiar pe
+            platforma noastră:
+          </p>
+          <LessonFigure
+            isVideo
+            withBorder
+            src="/videos/lessons/links-and-buttons/smooth-scrolling.mp4"
+            alt="Redarea comportamentului de smooth scrolling"
+          />
+        </section>
+        <section>
+          <LessonHeading as="h4" id="alte-exemple-unde-vom-folosi-linkuri">
+            Alte exemple unde vom folosi link-uri
+          </LessonHeading>
+          <p>
+            Sunt situații în care vrem să oferim unui user posibilitatea de a
+            deschide email-ul pentru a trimite un mesaj nou. În cazul acesta ne
+            vom folosi de
+            {' '}
+            <strong className="formatted">mailto</strong>
+            :
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="mailto:hello@frontend.ro">
+  Trimite email la Frontend.ro
+</a>`}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/mailto.png"
+            alt="Opțiunea mailto care ne permite să deschidem email-ul"
+            demo="/demo/html/mail-to"
+          />
+          <p>
+            Totodată, link-urile către numere de telefon sunt foarte uzuale mai
+            ales când navigăm de pe dispozitive mobile:
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="tel:+49.157.0156">+49 157 0156</a>
+<a href="tel:+1(555)5309">(555) 5309</a>
+`}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/tel.png"
+            alt="Link-uri către numere de telefon"
+            demo="/demo/html/linkuri-catre-numere-de-telefon"
+          />
+          <p>
+            Împreuna cu atributul
+            {' '}
+            <strong className="formatted">download</strong>
+            , putem instrui
+            browser-ul să downloadeze fișierul din cadrul link-ului în loc să îl
+            deschidă în pagina curentă:
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="/files/test.pdf" download>Download PDF</a>
+`}
+          />
+          <LessonFigure
```

Comment:
Inline demo. (pretty much inline all demos and possibly show videos for a couple that are more interesting. Having all those pictures with just text here takes up a lot of space and don't really do anything)